src/components/DroughtThresholds.vue

```diff
@@ -2904,7 +2906,7 @@ $usgsBlue: #032a56;
 }
 // currently empty scroll-by divs used to trigger animation
 .scrolly{
-  height:55vh;
+  height:85vh;
```

Comment:
Looking forward to testing beta to see if this helps with scrolling speed.

Reply:
Same! I'm thinking we may have to increase it even more, but we'll see